### PR TITLE
f-button@v0.3.0 - Update link button

### PR DESCRIPTION
**Pull request by FayeCarter**
### Changed
- `font-weight` and `text-decoration` for `:hover` states for link buttons.

### CODE REVIEW

**Review by ashleynolan — DISMISSED** [review]

File: packages/f-button/src/components/Button.vue [hunk 1]
```diff
@@ -326,13 +326,13 @@ $btn-sizeXSmall-lineHeight      : 1;
     padding: 0;
     margin: 0;
     color: $color-link-default;
-    font-weight: $font-weight-base;
-    text-decoration: underline;
+    font-weight: $font-weight-bold;
```
Comment (DamianMullins) [inline packages/f-button/src/components/Button.vue:329]: Quick question, the PR description says this is for large buttons but this looks like it's applied to all link buttons?

Comment (FayeCarter) [reply packages/f-button/src/components/Button.vue:329]: Ooh I'll change that, sorry. I should have deleted the word large. It's meant to be for all link buttons

**Comment by ashleynolan** [comment]
Have hidden the Danger check as version bump not needed for `f-searchbox`

**Review by ashleynolan — APPROVED** [review]

File: packages/f-searchbox/jest.config.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ module.exports = {
     moduleNameMapper: {
         '^@/(.*)$': '<rootDir>/src/$1',
         '^~include-media/(.*)$': '<rootDir>../../node_modules/include-media/$1',
-        '^~@justeat/(.*)$': '<rootDir>../../node_modules/@justeat/$1'
+        '^~@justeat/(.*)$': '<rootDir>../../node_modules/@justeat/$1',
+        '\\.(css|scss)$': 'jest-transform-stub'
```
Comment (kevinrodrigues) [inline packages/f-searchbox/jest.config.js:22]: 🎉

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**